### PR TITLE
docs: Creating form for auto-inviting to slack

### PR DESCRIPTION
**Pull request by captncraig**
This depends on a little script I made on hookscript.com which runs a little go script on a webhook. It keeps our access tokens secret and runs only when needed, using undocumented slack apis to do the magic.

### CODE REVIEW

File: docs/slackInvite.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+<p>Many bosun devs and users are usually availible on our slack channel. If you have any questions or issues this is the best place to 
+get help. Please fill out this form to automatically receive an invitation to join us:</p>
+
+	<div class="form-group">
+		<label for="email">Email address</label>
+		<input type="email" class="form-control" id="email" placeholder="Enter email">
+	</div>
+	<div class="form-group">
+		<label for="fname">First Name</label>
```
Comment (mjibson) [inline docs/slackInvite.html]: Do we really need first and last name? Can it work with just email address?

File: docs/slackInvite.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		<input type="text" class="form-control" id="fname" placeholder="First Name">
+	</div>
+	<div class="form-group">
+		<label for="lname">Last Name</label>
+		<input type="text" class="form-control" id="lname" placeholder="Last Name">
+	</div>
+  	<button id='submit' class="btn btn-default">Submit</button>
+	<div class="alert alert-success" role="alert" style='display:none'>Success! Check your email!</div>
```
Comment (mjibson) [inline docs/slackInvite.html]: double quotes on the style

File: docs/slackInvite.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	</div>
+	<div class="form-group">
+		<label for="fname">First Name</label>
+		<input type="text" class="form-control" id="fname" placeholder="First Name">
+	</div>
+	<div class="form-group">
+		<label for="lname">Last Name</label>
+		<input type="text" class="form-control" id="lname" placeholder="Last Name">
+	</div>
+  	<button id='submit' class="btn btn-default">Submit</button>
```
Comment (mjibson) [inline docs/slackInvite.html]: double quotes on the id

File: docs/slackInvite.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+		$.get(url, function(data){
+			$(".alert-success").show();
+			
+		}) .fail(function(jqXHR) {
+			$(".alert-danger").show();
+			$(".alert-danger").text(jqXHR.responseText);
+			$("#submit").prop('disabled', false);
+  		})
+	})
+
```
Comment (mjibson) [inline docs/slackInvite.html]: Remove this extra whitespace

File: docs/downloads.md [hunk 5]
```diff
@@ -22,6 +26,9 @@ title: Download Bosun
 			<li><a href="https://github.com/bosun-monitor/bosun/releases/download/{{relname}}/bosun-darwin-amd64"><strong>Mac</strong> amd64</a></li>
 			<li><a href="https://github.com/bosun-monitor/bosun/releases/download/{{relname}}/bosun-darwin-386"><strong>Mac</strong> 386</a></li>
 		</ul>
+{% else %}
+		<h3>View <a href="https://github.com/bosun-monitor/bosun/releases/latest">latest release</a> on github</h3>
```
Comment (mjibson) [inline docs/downloads.md]: github -> GitHub

File: docs/slackInvite.html (hunk 1 above)
Comment (captncraig) [inline docs/slackInvite.html]: It is the slack api that reuires first/last. Here is my code running on hookscript if you are curious:

https://gist.github.com/captncraig/d795ffa7990b09b596ed

File: docs/downloads.md [hunk 6]
```diff
@@ -3,12 +3,16 @@ layout: page
 title: Download Bosun
 ---
 
-{% assign releases = site.github.releases | where:"draft",false %}
-{% assign release = releases[0] %}
-{% assign relname = release.tag_name %}
+
```
Comment (mjibson) [inline docs/downloads.md]: Only one empty newline in a row, not three.

File: docs/slackInvite.html (hunk 1 above)
Comment (mjibson) [inline docs/slackInvite.html]: Can you show me documentation for that claim? I can't find users.admin.invite documented anywhere. And [this](https://github.com/outsideris/slack-invite-automation) project uses users.admin.invite with just email, no first or last name.

File: docs/scollector/index.html [hunk 7]
```diff
@@ -42,7 +42,9 @@
           <li><a href="https://github.com/bosun-monitor/bosun/releases/download/{{relname}}/scollector-linux-arm6"><strong>ARMv6</strong></a></li>
           <li><a href="https://github.com/bosun-monitor/bosun/releases/download/{{relname}}/scollector-linux-arm7"><strong>ARMv7</strong></a></li>
         </ul>
-
+		{% else %}
+			View <a href="https://github.com/bosun-monitor/bosun/releases/latest">latest release</a> on github
```
Comment (mjibson) [inline docs/scollector/index.html]: Was this also supposed to be in a separate PR?
